src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -62,7 +62,43 @@
     <MicrosoftDiaSymReaderNativeFileVersion>14.12.25830.2</MicrosoftDiaSymReaderNativeFileVersion>
   </PropertyGroup>
 
+  <Import Project="$(LibrariesProjectRoot)NetCoreAppLibrary.props" />
+
   <ItemGroup>
+    <!-- Libraries specified in NetCoreAppLibrary.props -->
+    <PlatformManifestFileEntry Include="@(NetCoreAppLibrary->'%(Identity).dll')" />
+    <PlatformManifestFileEntry Include="@(NetCoreAppLibraryNoReference->'%(Identity).dll')" />
+    <!-- Shim libraries -->
+    <PlatformManifestFileEntry Include="mscorlib.dll" />
+    <PlatformManifestFileEntry Include="netstandard.dll" />
+    <PlatformManifestFileEntry Include="WindowsBase.dll" />
+    <PlatformManifestFileEntry Include="Microsoft.VisualBasic.dll" />
+    <PlatformManifestFileEntry Include="System.ComponentModel.DataAnnotations.dll" />
```

Comment:
You can actually drive these off https://github.com/dotnet/runtime/blob/master/src/libraries/shims/netfxreference.props